extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -131,6 +131,13 @@ define('ext.wikia.adEngine.adContext', [
 			!noExternals
 		);
 
+		// Floating medrec
+		context.opts.floatingMedrec = !!(
+			context.opts.showAds && context.opts.adsInContent &&
+			(context.targeting.pageType === 'article' || context.targeting.pageType === 'search') &&
```

Comment:
We've got three conditions that includes `context.targeting.pageType === 'article'`. What do you think about creating a small helper function in `AdContext` called `isArticle()` and reusing it in these conditions?